go_api/server.go

```diff
@@ -8,7 +8,7 @@ import (
 	echoSwagger "github.com/swaggo/echo-swagger"
 
 	"github.com/labstack/echo/v4"
-	"gorm.io/driver/mysql"
+  "gorm.io/driver/mysql"
```

Comment:
おそらく昨日入れた自動成形を使えば
```suggestion
"gorm.io/driver/mysql"
```
スペースからタブになるはず